examples/go/hello-world/README.md

```diff
@@ -0,0 +1,86 @@
+# Hello, world!
+
+As customary, "hello, world!" is the simplest first app one would write, or extend.
+
+## What's in the App?
+
+This app:
+
+- Contains a [manifest](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=var%20Manifest)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/manifest/)), in
+  which it declares itself an HTTP app that acts as a bot, and binds to
+  locations in the user interface.
+- Contains a [send form](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=var%20SendForm)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/interactivity/))
+  that allows to enter a custom message.
+- [Binds](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=Bindinings%20callback)
+  ([docs](https://developers.mattermost.com/integrate/apps/api/bindings/)) the
+  form to the channel header, and defines a `/helloworld send` command.
+- Contains the [Send](https://github.com/mattermost/mattermost-plugin-apps/blob/master/examples/go/hello-world/hello.go#:~:text=func%20Send)
+  call ([docs](https://developers.mattermost.com/integrate/apps/api/call/)) handler that sends a parameterized message back to the user.
+
+## Prerequisites
+
+Before you can start with your app, you first need to set up a local developer
+environment following the [server](https://developers.mattermost.com/contribute/server/developer-setup/)
+and [webapp](https://developers.mattermost.com/contribute/webapp/developer-setup/)
+setup guides. You need Mattermost v6.6 or later.
+
+In the System Console, ensure that the following are set to **true**:
+
+- `Enable Bot Account Creation`
+- `Enable OAuth 2.0 Service Provider`
+
+You also need at least `go1.16` installed. Please follow the guide
+[here](https://golang.org/doc/install) to install the latest version.
+
+#### Install the Apps plugin
+
+The [apps plugin](https://github.com/mattermost/mattermost-plugin-apps) is a
+communication bridge between your app and the Mattermost server. To install it
+on your local server, start by cloning the code in a directory of your choice
+run:
+
+```bash
+git clone https://github.com/mattermost/mattermost-plugin-apps.git
+```
+
+Then build the plugin using:
+
+```bash
+cd mattermost-plugin-apps
+make dist
+```
+
+Then upload it to your local Mattermost server via the System Console.
```

Comment:
@levb - Before the system integrator or developer can upload the apps plugin to their local MM server, they need to enable plugin uploads (via the config.json file). This step is buried in the Using and Managing Plugins > [Custom Plugins](https://developers.mattermost.com/integrate/admin-guide/admin-plugins-beta/#custom-plugins) developer documentation currently. Given that this is a required step for success, is there a way to pre-set this so that the SI/dev doesn't need to manage this manually? (If that's not possible, we need to ensure this step is clearly documented up front and easy to find).